configure.ac

```diff
@@ -289,6 +289,14 @@ AM_COND_IF(BUILD_MODULE_RELAY,
            AC_DEFINE([HAVE_MODULE_RELAY], [1],
                      [Compile with Relay feature support]))
 
+dnl Dircache module.  (This cannot be enabled or disabled independently of
```

Comment:
Please add a note that this module isn't listed in `--list-modules`.

Reply:
We also don't have a `have_module_dircache()` function. I don't mind either way, but we should either have the function, or have a comment in `dircache.h` explaining why it doesn't exist.

We don't check for dircache in `list_enabled_modules()`. That's actually a good thing, because if we did, we'd have to make a large number of changes to the test cases in `test_parseconf.sh`. Let's add comments in `list_enabled_modules()` and `test_parseconf.sh` explaining why the dircache module isn't listed.

If you do want to list the dircache module, please make test_parseconf.sh ignore the dircache module, so we don't have to modify a whole bunch of tests. (By adding `dircache_` to the `relay_` disabled module file names.)